bluetti_mqtt/mqtt_client.py

```diff
@@ -559,7 +559,7 @@ async def _init_device(self, device: BluettiDevice, client: Client):
         def payload(id: str, device: BluettiDevice, field: MqttFieldConfig) -> str:
             ha_id = id if not field.id_override else field.id_override
             payload_dict = {
-                'state_topic': f'bluetti/state/{device.type}-{device.sn}/{id}',
+                'state_topic': f'custom-data/bluetti/state/{device.type}-{device.sn}/{id}',
```

Comment:
This change should at least be in a separate commit, even better in a separate PR. This way we can try to upstream the AC200L support.

Reply:
Additionally I would suggest to have a CLI option to define an MQTT topic prefix. That way anybody can use it for whatever MQTT topic structure they have.

Reply:
So now the default topic is: (for e.g custom-data/bluetti/state/AC200L-XXXXXX)

Do you suggest to modify it to be able to change the whole structure of the TOPIC or just the first prefix (i.e **custom-data**).

From my point of view we should at least  keep the vendor part bluetti + {device.type}-{device.sn}/{id}  unchangeable since we have more than one station and we need to indicate the vendor.


So my suggestion is to adapt it to only change the first prefix.

Reply:
Only the prefix